app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/views/ProvideEscalatedToForGrantingPage.scala.html

```diff
@@ -0,0 +1,54 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.GrantedJourneyController.{ProvideEscalatedToForm, ViewModel}
+@import uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.models._
+@import uk.gov.hmrc.apiplatform.modules.submissions.domain.models._
+
+@this(layout: Layout, breadcrumbs: Breadcrumbs)
+
+@(form: Form[ProvideEscalatedToForm], model: ViewModel)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = Some(messages("escalatedtouser.pagetitle")), beforeContent = Some(breadcrumbs(None))) {
+  <link rel="stylesheet" href='@routes.Assets.versioned("stylesheets/application.css")'>
+
+  @ErrorSummary(form)
+  
+  @helper.form(action = uk.gov.hmrc.apigatekeeperapprovalsfrontend.controllers.routes.GrantedJourneyController.provideEscalatedToAction(model.applicationId), 'class -> "form") {
+    @helper.CSRF.formField
+
+    <span class="govuk-caption-l">@model.appName</span>
+    <h1 class="govuk-label-wrapper">
+      <label class="govuk-label govuk-label--l" for="first-name">
+        @{messages("escalatedtouser.title")}
+      </label>
+    </h1>
+    <div id="escalatedbyuser-hint" class="govuk-hint">
+      @{messages("escalatedtouser.note")}
+    </div>
+
+    <div class="govuk-form-group">
+      <label class="govuk-label" for="escalatedto">
+        @{messages("escalatedtouser.escalatedto")}
+      </label>
+      <input class="govuk-input govuk-input--width-10" id="escalatedto" name="escalatedto" type="text">
```

Comment:
should the form have 2 fields for the name first/last ?

Reply:
Yes, long story but fixed now....